bindings/python/tests/compiler_tf_test.py

```diff
@@ -20,9 +20,9 @@
 
 # TODO: No idea why pytype cannot find names from this module.
 # pytype: disable=name-error
```

Comment:
Any chance this goes away? Doesn't need to be this PR, but could be a nice follow-up to see if some of these disables can now be removed.

Reply:
No, but I _did_ find another error that gets raised with my local (up-to-date) version of `pytype`, so I added another disable statement.

I think that `pytype` provides less value than it gives. We need to make dozens of exceptions for it in our code, and need an overly complicated `.sh` script to even get it to vaguely work. Nice in theory, but awful in practice.

The new local error:

```
[13/24] check compiler_xla_test
FAILED: /usr/local/google/home/meadowlark/dev/iree/.pytype/pyi/compiler_xla_test.pyi 
/usr/local/google/home/meadowlark/dev/iree/.venv/iree-all/bin/python3 -m pytype.single --disable import-error,pyi-error --imports_info /usr/local/google/home/meadowlark/dev/iree/.pytype/imports/compiler_xla_test.imports --module-name compiler_xla_test -V 3.8 -o /usr/local/google/home/meadowlark/dev/iree/.pytype/pyi/compiler_xla_test.pyi --analyze-annotated --nofail --quick /usr/local/google/home/meadowlark/dev/iree/bindings/python/tests/compiler_xla_test.py
File "/usr/local/google/home/meadowlark/dev/iree/bindings/python/tests/compiler_xla_test.py", line 73, in testCompileBinaryPbFileOutputFile: No attribute 'DEFAULT_TESTING_BACKENDS' on module 'iree.compiler' [module-attr]
```